scripts/socketplane.sh

```diff
@@ -316,6 +320,12 @@ start_socketplane() {
 
     mkdir -p /var/run/socketplane
     echo $cid > /var/run/socketplane/cid
+
+    if [ "$ps" = "yes" ]; then
+	pscid=$(docker run -d --name powerstrip -v /var/run/docker.sock:/var/run/docker.sock \
+	    -v /etc/socketplane/adapters.yml:/etc/powerstrip/adapters.yml --net=host  \
```

Comment:
Ya. We could state that .yml file should be in /etc/socketplane/adapters.yml.
But, there are 2 cases we can handle it automatically.
* Developer machine. We could potentially copy the adapters.yml in the local directory and copy it to /etc/socketplane
* if the users uses the install.sh (as specified in the README.md), then we can fetch the adapters.yml from the github repo. (We do the same today for socketplane.sh and place it in /usr/bin)